android/src/main/java/com/applovin/reactnative/AppLovinMAXAdViewManager.java

```diff
@@ -84,6 +84,18 @@ public void setAdFormat(final AppLovinMAXAdView view, final String adFormatStr)
         view.setAdFormat( adFormatStr );
     }
 
+    @ReactProp(name = "extraParameters")
+    public void setExtraParameters(final AppLovinMAXAdView view, @Nullable final ReadableMap value)
+    {
+        view.setExtraParameters( value );
+    }
+
+    @ReactProp(name = "localExtraParameters")
+    public void setLocalExtraParameters(final AppLovinMAXAdView view, @Nullable final ReadableMap value)
+    {
+        view.setLocalExtraParameters( value );
+    }
+
```

Comment:
nit but we should order the methods in the adview manager the same on both platforms. Like on Android we ordered it as:

- setPlacement
- setCustomData
- setAdaptiveBannerEnabled
- setAutoRefresh
- setAdUnitId
- setAdFormat
- setExtraParameter
- setLocalExtraParameter

reads better, and its free to organize as such.

Reply:
Rearranged for both AdView and NativeAdView.